Nodejs/Product/Nodejs/Extensions.cs

```diff
@@ -121,10 +121,12 @@ out project
             }
         }
 
-        internal static T[] Append<T>(this T[] list, T item) {
-            T[] res = new T[list.Length + 1];
+        internal static T[] Append<T>(this T[] list, params T[] items) {
```

Comment:
I'm not quite sure where this is used, but it might be worth adding a new overload rather then just changing this one.  If it's perf critical you're now creating an extra new array whenever you need to append a single item.